app/views/facility_accounts/show_statement_list.html.haml

```diff
@@ -1,29 +1,29 @@
 = content_for :sidebar do
-  = render :partial => 'admin/shared/sidenav_billing', :locals => { :sidenav_tab => 'accounts' }
+  = render "admin/shared/sidenav_users", sidenav_tab: "accounts"
 
 = content_for :tabnav do
-  = render :partial => 'admin/shared/tabnav_payment_method', :locals => {:secondary_tab => 'statements'}
+  = render "admin/shared/tabnav_payment_method", secondary_tab: "statements"
 
 = content_for :h1 do
-  =h current_facility
+  = current_facility
 
 %h2= @account
 
 - if @statements.empty?
-  %p.notice= t('.notice')
+  %p.notice= t(".notice")
 - else
   %table.table.table-striped.table-hover
     %thead
       %tr
         %th
-        %th= t('.th.sent_at')
-        %th= t('.th.sent_by')
-        %th.currency= t('.th.orders')
+        %th= t(".th.sent_at")
+        %th= t(".th.sent_by")
+        %th.currency= t(".th.orders")
     %tbody
-      - @statements.each do |s|
+      - StatementPresenter.wrap(@statements).each do |statement|
```

Comment:
I like this change. I'm just curious how this is related to the moving of the tab?